src/NServiceBus.RavenDB/Internal/DocumentStoreInitializer.cs

```diff
@@ -116,17 +118,22 @@ void ApplyConventions(IReadOnlySettings settings)
             }
         }
 
-        static void EnsureClusterConfiguration(IDocumentStore store)
+        static void EnsureClusterConfiguration(IDocumentStore store, bool useClusterWideTransactions)
         {
             using (var s = store.OpenSession())
             {
-                var getTopologyCmd = new GetDatabaseTopologyCommand();
-                s.Advanced.RequestExecutor.Execute(getTopologyCmd, s.Advanced.Context);
+                var databaseTopology = new GetDatabaseTopologyCommand();
+                s.Advanced.RequestExecutor.Execute(databaseTopology, s.Advanced.Context);
+                var clusterTopology = new GetClusterTopologyCommand();
+                s.Advanced.RequestExecutor.Execute(clusterTopology, s.Advanced.Context);
 
-                // Currently do not support clusters.
-                if (getTopologyCmd.Result.Nodes.Count != 1)
+                if (useClusterWideTransactions && databaseTopology.Result.Nodes.Count == 1 && clusterTopology.Result.Topology.AllNodes.Count > 1)
                 {
-                    throw new InvalidOperationException("RavenDB Persistence does not support database groups with multiple database nodes. Only single-node configurations are supported.");
+                    Logger.Warn($"The replication factor of the configured database is 1, while more nodes were detected in the cluster. If the intent is to keep the replication factor as configured, do not enable {nameof(RavenDbSettingsExtensions.EnableClusterWideTransactions)} on the persistence configuration");
```

Comment:
```suggestion
                    Logger.Warn($"The replication factor of the configured database is 1, while more nodes were detected in the cluster. If the intent is to keep the replication factor as configured, do not enable {nameof(RavenDbSettingsExtensions.EnableClusterWideTransactions)} on the persistence configuration since it comes with performance implications.");
```